assets/airports/kmia.json

```diff
@@ -45,377 +45,986 @@
+            },
+            "body": ["HILEY", "KAINS", "CIMBA", "JESSS", "RUBOE", "#271"],
+            "rwy": {
+                "8L": [],
+                "8R": [],
+                "9": [],
+                "12": [],
+                "26L": [],
+                "26R": [],
+                "27": [],
+                "30": []
             },
-            "body": ["HILEY", "KAINS", "CIMBA", "JESSS", "RUBOE"],
-            "draw": [["OMN*","MALET","PCMAN","MLB","TRV","PHORD","SMUGS","MORGA","PBI","BTOGA","HILEY"], ["JORAY*","OSOGY","AMZAR","STAPL","MILSY","BOYUR","HILEY"], ["ZFP*","STAPL"], ["HILEY","KAINS","CIMBA","JESSS","RUBOE"]]
+            "draw": [
+                ["JORAY*", "OSOGY", "AMZAR", "STAPL", "MILSY", "BOYUR", "HILEY", "KAINS", "CIMBA", "JESSS", "RUBOE"],
+                ["OMN*", "MALET", "PCMAN", "MLB", "TRV", "PHORD", "SMUGS", "MORGA", "PBI", "BTOGA", "HILEY"],
+                ["ZFP*", "STAPL"]
+            ]
         },
         "SSCOT3": {
             "icao": "SSCOT3",
             "name": "Scott Three",
             "entryPoints": {
-                "BAGGS": [["BAGGS", "S280"], "BAARY"],
-                "SZW": ["SZW", "HEVVN", "LEGGT", "PLYER", "SEATE", "JUULI", "BAARY"],
-                "OTK": ["OTK", "HITTR", "TUCKE", "JUULI", "BAARY"],
-                "TAY": ["TAY", "HITTR", "TUCKE", "JUULI", "BAARY"]
+                "BAGGS": ["BAGGS"],
+                "OTK": ["OTK", "HITTR", "TUCKE", "JUULI"],
+                "SZW": ["SZW", "HEVVN", "LEGGT", "PLYER", "SEATE", "JUULI"],
+                "TAY": ["TAY", "HITTR", "TUCKE", "JUULI"]
+            },
+            "body": ["BAARY", "CYY", "DEEDS", "SSCOT", "RUBOE", "WALIP", "LECIT", "#126"],
+            "rwy": {
+                "8L": [],
+                "8R": [],
+                "9": [],
+                "12": [],
+                "26L": [],
+                "26R": [],
+                "27": [],
+                "30": []
             },
-            "body": ["BAARY", "CYY", ["DEEDS", "A160|S280"], ["SSCOT", "A100"], "RUBOE", "WALIP"],
-            "draw": [["BAGGS*","BAARY"], ["SZW*","HEVVN","LEGGT","PLYER","SEATE","JUULI","BAARY"], ["OTK*","HITTR","TUCKE","JUULI","BAARY"], ["TAY*","HITTR","TUCKE","JUULI","BAARY"], ["BAARY","CYY","DEEDS","SSCOT","RUBOE","WALIP"]]
+            "draw": [
+                ["BAGGS*", "BAARY", "CYY", "DEEDS", "SSCOT", "RUBOE", "WALIP", "LECIT"],
+                ["OTK*", "HITTR", "TUCKE", "JUULI", "BAARY"],
+                ["SZW", "HEVVN", "LEGGT", "PLYER", "SEATE", "JUULI"],
+                ["TAY*", "HITTR"]
+            ]
         }
     },
     "spawnPatterns": [
         {
             "origin": "kmia",
             "destination": "",
             "category": "departure",
-            "route": "KMIA.WINCO2.WINCO",
+            "route": "KMIA.EONNS2.EONNS..ELLEE..URSUS..GELOG",
+            "method": "random",
+            "rate": 3,
+            "airlines": [
+                ["aal", 10],
```

Comment:
Could we make the list of airlines longer? It's somewhat unrealistic with just 5 airlines

Reply:
ah! yes, i meant to do that, with relevant airlines from relevant directions. Thanks for the reminder 😄

Reply:
done